src/job_worker/Jobs.jl

```diff
@@ -0,0 +1,226 @@
+using JSON3
+using Logging
+using Dates
+
+#
+# Core type definitions
+#
```

Comment:
If this indicates what the file is for, I'd prefer this to be in a docstring at top of file.

Reply:
resolved